Location: UncheckedIOExceptionsTest.java

Content: 

@Test public void testCreateWithException(){

  final Object message1="test1";

  final Object message2="test2";

  final IOException ioe=new IOException(message2.toString());

  try {

    throw UncheckedIOExceptions.create(message1,ioe);

  }

 catch (  final UncheckedIOException e) {

    assertEquals(message1,e.getMessage());

    assertEquals(message2,e.getCause().getMessage());

  }

}

